core/src/main/scala/akka/kafka/ConsumerSettings.scala

```
@@ -133,7 +182,7 @@ object ConsumerSettings {
     keyDeserializer: Deserializer[K],
     valueDeserializer: Deserializer[V]
   ): ConsumerSettings[K, V] =
-    apply(system, keyDeserializer, valueDeserializer)
+    apply(system, Option(keyDeserializer), Option(valueDeserializer))
```

will this `Option` hide an error when I pass null in serializers?